_data/core.yml

```diff
@@ -0,0 +1,71 @@
+coredevs:
+  - first: Stefani
+    last: Lumnitz
+    avatar: https://avatars2.githubusercontent.com/u/35147471?v=4&s=400 
+    affiliation: University of British Columbia
+    gh_handle: slumnitz
+  - first: Levi
+    last: Wolf  
+    avatar: https://avatars3.githubusercontent.com/u/2250995?v=4&s=400
+    affiliation: University of Bristol
+    gh_handle: ljwolf
+  - first: Pedro
+    last: Amaral
+    avatar: https://avatars0.githubusercontent.com/u/4238922?v=4&s=400
+    affiliation: Universidade Federal de Minas Gerais
+    gh_handle: pedrovma 
+  - first: Dani
+    last: Arribas-Bel
+    avatar: https://avatars3.githubusercontent.com/u/417363?v=4&s=400
+    affiliation: University of Liverpool
+    gh_handle: darribas
+  - first: Serge
```

Comment:
The name of the affiliation appears next to the name, not underneath for Serge, Wei, Levi and Hu